src/main/java/online/kingdomkeys/kingdomkeys/client/gui/synthesis/ShopScreen.java

```diff
@@ -289,8 +289,8 @@ protected void renderSelectedData(GuiGraphics gui, int mouseX, int mouseY, float
 				gui.drawString(minecraft.font, Utils.translateToLocal(Strings.Gui_Shop_Buy_Cost)+":", 2, -20, Color.yellow.getRGB());
 				String line = item.getCost()+" "+Utils.translateToLocal(Strings.Gui_Menu_Main_Munny);
 				gui.drawString(minecraft.font, line, boxM.getWidth() - minecraft.font.width(line) - 10, -20, item.getCost() > playerData.getMunny() ? Color.RED.getRGB() : Color.GREEN.getRGB());
-				gui.drawString(minecraft.font, Utils.translateToLocal("Tier")+":", 2, -10, Color.yellow.getRGB());
-				line = Utils.getTierFromInt(item.getTier())+" "+(10 + item.getTier()*2)+"exp";
+				gui.drawString(minecraft.font, Gui_Shop_Buy_Tier+":", 2, -10, Color.yellow.getRGB());
```

Comment:
Utils.translateToLocal(Strings.Gui_Shop_Buy_Tier)
We probably could make this include the colon in the translation too